### ISSUE TITLE
 Request for validation set list

### ISSUE THREAD

**Issue by cikitime** [uni-medical/SAM-Med3D#47]
I hope you're doing well. I recently read your paper and found it to be very informative and valuable.

I noticed that the validation set list was not provided in the paper or the supplementary materials. Would it be possible for you to share the list of samples used in the validation set? Knowing this information helps me conduct a comparative experiment.

Thank you for your time.

**Comment by cikitime** [comment]
thank you！

**Comment by joseph-bae** [comment]
Hello,

I have a data leakage concern. The case list provided contains the entirety of the BraTS 2021 patient list, despite BraTS 2021 also being listed in the train set. Is this an error? Do you have the list of BraTS patients used for training and testing independently?

**Comment by blueyo0** [comment]
> Hello,
> 
> I have a data leakage concern. The case list provided contains the entirety of the BraTS 2021 patient list, despite BraTS 2021 also being listed in the train set. Is this an error? Do you have the list of BraTS patients used for training and testing independently?

Hi, @joseph-bae Mayde here's some misunderstanding. We used the train/val set of BraTS 2021 for our training and validation, just like many other works that trained and validated on BraTS. I think there is no data leakage issue.
